1.9/overview/architecture/task-types.md

```diff
@@ -0,0 +1,49 @@
+---
+post_title: Task Types
+nav_title: Task Types
+menu_order: 2
+---
+
+DC/OS is capable of running many different kinds of workloads. All workloads are composed of tasks.
```

Comment:
Suggest: `DC/OS can run many different kinds of workloads, which are composed of tasks.`